### PR TITLE
Added a selection (day/week/month/year)

### PR DESCRIPTION
**Pull request by SN33DS**

### CODE REVIEW

File: StealMemes.py [hunk 1]
```diff
@@ -10,13 +11,19 @@
 from selenium.common.exceptions import TimeoutException
 from selenium.webdriver.support.ui import WebDriverWait
 
+
+os.system('cls')
```
Comment (YulkyTulky) [inline StealMemes.py:15]: this is not cross-platform, an ANSI escape sequence would be better

File: StealMemes.py [hunk 2]
```diff
@@ -10,13 +11,19 @@
...
+Category = sys.argv[1] # (either day, week, month or year)
```
Comment (YulkyTulky) [inline StealMemes.py:16]: good addition, just needs to include error handling if user enters something besides day, week, month, or year (or enters nothing at all)

File: StealMemes.py [hunk 3]
```diff
@@ -10,13 +11,19 @@
...
+
+folder = f'Top {Category} memes'
```
Comment (YulkyTulky) [inline StealMemes.py:18]: I think a better naming strategy is something like this:
Day - 4.29.20
or
Week - 4.26.20
or
Year - 1.1.20

Comment (SN33DS) [reply StealMemes.py:18]: You could always use datetime module to insert current date in the file name (more or less precise)

File: StealMemes.py (hunk 2 above)
Comment (SN33DS) [reply StealMemes.py:16]: Will look into this

**Comment by YulkyTulky** [comment]
I will implement some of these features!

File: StealMemes.py (hunk 3 above)
Comment (SN33DS) [reply StealMemes.py:18]: The ANSI escape code would be 'CSI 2 J'

File: StealMemes.py (hunk 1 above)
Comment (SN33DS) [reply StealMemes.py:15]: The ANSI escape code would be 'CSI 2 J'

File: StealMemes.py (hunk 3 above)
Comment (YulkyTulky) [reply StealMemes.py:18]: I have updated the project using your suggestions! It'd be very helpful if you could test it as I'm not sure my implementation to clear the screen will work on windows

Comment (YulkyTulky) [reply StealMemes.py:18]: > You could always use datetime module to insert current date in the file name (more or less precise)
I have updated the project using your suggestions! It'd be very helpful if you could test it as I'm not sure my implementation to clear the screen will work on windows

Comment (SN33DS) [reply StealMemes.py:18]: I tested your way of clearing the screen but it didn't work though I found 'print(chr(27) + "[2J")' worked

Comment (YulkyTulky) [reply StealMemes.py:18]: > I tested your way of clearing the screen but it didn't work though I found 'print(chr(27) + "[2J")' worked
that didn't work perfectly on my machine. I've instead done this:
```os.system('cls' if os.name == 'nt' else 'clear')```

Comment (SN33DS) [reply StealMemes.py:18]: That works fine for me as well

Comment (SN33DS) [reply StealMemes.py:18]: Its really starting to look like a complete tool

Comment (YulkyTulky) [reply StealMemes.py:18]: > Its really starting to look like a complete tool
thanks, haha! please make a pull request/give suggestions if u have any!

Comment (SN33DS) [reply StealMemes.py:18]: I'm afraid that's all I have for now :)

**Pull request closed without merging**